internal/spokes/spokes.go

```diff
@@ -251,6 +254,18 @@ func (r *SpokesReceivePack) performReferenceDiscovery(ctx context.Context) error
 	return nil
 }
 
+func (r *SpokesReceivePack) getHiddenRefs() ([]string, error) {
+	config, err := config.GetConfig(r.repoPath, "receive.hiderefs")
```

Comment:
Should this also check `transfer.hiderefs`? It looks like babeld will set `transfer.hiderefs` [here](https://github.com/github/babeld/blob/89fc82eff6e064dedf23206811c18ea074909ef8/src/util.c#L839) and [here](https://github.com/github/babeld/blob/89fc82eff6e064dedf23206811c18ea074909ef8/src/util.c#L866).

Reply:
Added it here 4d99f7945242c36e23c560e627f3d3802fe0f6bf to the list of hidden refs